docs pipeline fail

### What's failing?

Documentation build (Docusaurus)

### Branch

main

### GitHub Actions Run Link

https://github.com/coding-kitties/PyIndicators/actions/runs/22548284347/job/65314131726

### Error Output

```shell
Run mkdir -p docs/static/img/indicators
cp: 'static/images/indicators/accumulation_distribution_zones.png' and 'docs/static/img/indicators/accumulation_distribution_zones.png' are the same file
cp: 'static/images/indicators/adx.png' and 'docs/static/img/indicators/adx.png' are the same file
cp: 'static/images/indicators/atr.png' and 'docs/static/img/indicators/atr.png' are the same file
cp: 'static/images/indicators/bearish_divergence.png' and 'docs/static/img/indicators/bearish_divergence.png' are the same file
cp: 'static/images/indicators/bollinger_bands.png' and 'docs/static/img/indicators/bollinger_bands.png' are the same file
cp: 'static/images/indicators/bollinger_overshoot.png' and 'docs/static/img/indicators/bollinger_overshoot.png' are the same file
cp: 'static/images/indicators/breaker_blocks.png' and 'docs/static/img/indicators/breaker_blocks.png' are the same file
cp: 'static/images/indicators/bullish_divergence.png' and 'docs/static/img/indicators/bullish_divergence.png' are the same file
cp: 'static/images/indicators/buy_side_sell_side_liquidity.png' and 'docs/static/img/indicators/buy_side_sell_side_liquidity.png' are the same file
cp: 'static/images/indicators/crossover.png' and 'docs/static/img/indicators/crossover.png' are the same file
cp: 'static/images/indicators/crossunder.png' and 'docs/static/img/indicators/crossunder.png' are the same file
cp: 'static/images/indicators/detect_peaks.png' and 'docs/static/img/indicators/detect_peaks.png' are the same file
cp: 'static/images/indicators/ema.png' and 'docs/static/img/indicators/ema.png' are the same file
cp: 'static/images/indicators/ema_trend_ribbon.png' and 'docs/static/img/indicators/ema_trend_ribbon.png' are the same file
cp: 'static/images/indicators/fair_value_gap.png' and 'docs/static/img/indicators/fair_value_gap.png' are the same file
cp: 'static/images/indicators/fibonacci_retracement.png' and 'docs/static/img/indicators/fibonacci_retracement.png' are the same file
cp: 'static/images/indicators/golden_zone.png' and 'docs/static/img/indicators/golden_zone.png' are the same file
cp: 'static/images/indicators/golden_zone_signal.png' and 'docs/static/img/indicators/golden_zone_signal.png' are the same file
cp: 'static/images/indicators/has_any_lower_then_threshold.png' and 'docs/static/img/indicators/has_any_lower_then_threshold.png' are the same file
cp: 'static/images/indicators/internal_external_liquidity_zones.png' and 'docs/static/img/indicators/internal_external_liquidity_zones.png' are the same file
cp: 'static/images/indicators/internal_external_liquidity_zones_signal.png' and 'docs/static/img/indicators/internal_external_liquidity_zones_signal.png' are the same file
cp: 'static/images/indicators/liquidity_levels_voids.png' and 'docs/static/img/indicators/liquidity_levels_voids.png' are the same file
cp: 'static/images/indicators/liquidity_pools.png' and 'docs/static/img/indicators/liquidity_pools.png' are the same file
cp: 'static/images/indicators/liquidity_sweeps.png' and 'docs/static/img/indicators/liquidity_sweeps.png' are the same file
cp: 'static/images/indicators/macd.png' and 'docs/static/img/indicators/macd.png' are the same file
cp: 'static/images/indicators/market_structure_choch_bos.png' and 'docs/static/img/indicators/market_structure_choch_bos.png' are the same file
cp: 'static/images/indicators/market_structure_ob.png' and 'docs/static/img/indicators/market_structure_ob.png' are the same file
cp: 'static/images/indicators/mitigation_blocks.png' and 'docs/static/img/indicators/mitigation_blocks.png' are the same file
cp: 'static/images/indicators/momentum_confluence.png' and 'docs/static/img/indicators/momentum_confluence.png' are the same file
cp: 'static/images/indicators/momentum_cycle_sentry.png' and 'docs/static/img/indicators/momentum_cycle_sentry.png' are the same file
cp: 'static/images/indicators/moving_average_envelope.png' and 'docs/static/img/indicators/moving_average_envelope.png' are the same file
cp: 'static/images/indicators/nadaraya_watson_envelope.png' and 'docs/static/img/indicators/nadaraya_watson_envelope.png' are the same file
cp: 'static/images/indicators/opening_gap.png' and 'docs/static/img/indicators/opening_gap.png' are the same file
cp: 'static/images/indicators/optimal_trade_entry.png' and 'docs/static/img/indicators/optimal_trade_entry.png' are the same file
cp: 'static/images/indicators/order_blocks.png' and 'docs/static/img/indicators/order_blocks.png' are the same file
cp: 'static/images/indicators/premium_discount_zones.png' and 'docs/static/img/indicators/premium_discount_zones.png' are the same file
cp: 'static/images/indicators/price_action_combined.png' and 'docs/static/img/indicators/price_action_combined.png' are the same file
cp: 'static/images/indicators/price_action_confluence.png' and 'docs/static/img/indicators/price_action_confluence.png' are the same file
cp: 'static/images/indicators/pulse_mean_accelerator.png' and 'docs/static/img/indicators/pulse_mean_accelerator.png' are the same file
cp: 'static/images/indicators/pure_price_action_liquidity_sweeps.png' and 'docs/static/img/indicators/pure_price_action_liquidity_sweeps.png' are the same file
cp: 'static/images/indicators/range_intelligence.png' and 'docs/static/img/indicators/range_intelligence.png' are the same file
cp: 'static/images/indicators/rejection_blocks.png' and 'docs/static/img/indicators/rejection_blocks.png' are the same file
cp: 'static/images/indicators/rsi.png' and 'docs/static/img/indicators/rsi.png' are the same file
cp: 'static/images/indicators/sma.png' and 'docs/static/img/indicators/sma.png' are the same file
cp: 'static/images/indicators/sto.png' and 'docs/static/img/indicators/sto.png' are the same file
cp: 'static/images/indicators/strong_weak_high_low.png' and 'docs/static/img/indicators/strong_weak_high_low.png' are the same file
cp: 'static/images/indicators/supertrend.png' and 'docs/static/img/indicators/supertrend.png' are the same file
cp: 'static/images/indicators/supertrend_basic.png' and 'docs/static/img/indicators/supertrend_basic.png' are the same file
cp: 'static/images/indicators/supertrend_clustering.png' and 'docs/static/img/indicators/supertrend_clustering.png' are the same file
cp: 'static/images/indicators/trendline_breakout_navigator.png' and 'docs/static/img/indicators/trendline_breakout_navigator.png' are the same file
cp: 'static/images/indicators/volume_imbalance.png' and 'docs/static/img/indicators/volume_imbalance.png' are the same file
cp: 'static/images/indicators/volume_weighted_trend.png' and 'docs/static/img/indicators/volume_weighted_trend.png' are the same file
cp: 'static/images/indicators/volumetric_supply_demand_zones.png' and 'docs/static/img/indicators/volumetric_supply_demand_zones.png' are the same file
cp: 'static/images/indicators/wilders_rsi.png' and 'docs/static/img/indicators/wilders_rsi.png' are the same file
cp: 'static/images/indicators/willr.png' and 'docs/static/img/indicators/willr.png' are the same file
cp: 'static/images/indicators/wma.png' and 'docs/static/img/indicators/wma.png' are the same file
cp: 'static/images/indicators/z_score_predictive_zones.png' and 'docs/static/img/indicators/z_score_predictive_zones.png' are the same file
cp: 'static/images/indicators/zero_lag_ema_envelope.png' and 'docs/static/img/indicators/zero_lag_ema_envelope.png' are the same file
Error: Process completed with exit code 1.
```

### Steps to Reproduce

_No response_

### Recent Changes

_No response_

### Severity

🔴 Blocking — main branch is broken, no PRs can merge

### Deliverables

- [ ] Fix the failing test(s)
- [ ] Fix the build/lint issue
- [ ] Add missing test coverage to prevent regression
- [ ] Update CI workflow configuration

### Additional Context

_No response_

--- a/.github/workflows/deploy-docs.yml
+++ b/.github/workflows/deploy-docs.yml
@@ -47,12 +47,6 @@ jobs:
       - name: Generate doc pages
         run: python scripts/generate_docs.py
 
-      # ── Copy images to Docusaurus static dir ──
-      - name: Copy indicator images to docs
-        run: |
-          mkdir -p docs/static/img/indicators
-          cp static/images/indicators/*.png docs/static/img/indicators/
-
       # ── Node: build Docusaurus ──
       - name: Set up Node.js
         uses: actions/setup-node@v4

